dev-util/intel-graphics-compiler/files/intel-graphics-compiler-1.0.8173-fix-missing-limits.patch

```diff
@@ -0,0 +1,20 @@
+From 8e1a461d3e6b85a6cf018caf6abf4a3ba9a1758d Mon Sep 17 00:00:00 2001
```

Comment:
[Ideally](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) link to where patches come from, but it's not a big deal for this. Also, briefly, why. Any bugs, etc.

(Applies for all.)

Reply:
This one is an upstream PR, I have added the information and a link.